src/net/socket_state.rs

```diff
@@ -160,11 +158,4 @@ mod test {
         let addr = format!("{}:{}", TEST_BAD_HOST_IP, TEST_PORT).to_socket_addrs();
         assert!(addr.is_err());
     }
-
-    #[test]
-    fn test_poll_for_invalid_clients() {
```

Comment:
removing test because they fail :+1:. Actually don't think this test adds much value. So it's fine with me

Reply:
This ond didn't do anything. There's another one that fails, need to check why. For this test I'm gonna add a better one.